src/Smart.Design.Razor/TagHelpers/Header/HeaderHtmlGenerator.cs

```diff
@@ -34,6 +34,12 @@ public HeaderHtmlGenerator(IIconHtmlGenerator iconHtmlGenerator)
             Thread.CurrentThread.CurrentUICulture = CultureInfo.CreateSpecificCulture(targetLanguage);
         }
 
+        var language = (targetLanguage?.ToUpper()) switch
+        {
+            "FR" => "Français",
+            "NL" => "Nederlands",
+            _ => "English",
+        };
```

Comment:
Est-ce qu'on générerait pas la culture du thread dans tous les cas (il en faut toujours une et on est pas sûr que par défaut le thread de l'application soit en anglais vu que ça dépend de l'hôte et des paramètres de l'app qui va utiliser le composant)?
Peut-être un truc du genre?

```suggestion
          (string CultureCode, string Language) languageCulture = targetLanguage?.ToUpper() switch
        {
            "FR" => ("FR", "Français"),
            "NL" => ("NL", "Nederlands"),
            _ => ("EN", "English")
        };

        Thread.CurrentThread.CurrentUICulture = CultureInfo.CreateSpecificCulture(languageCulture.CultureCode);
```